frontend/packages/ux-editor/src/components/config/componentSpecificContent/FileUpload/FileUploadComponent.tsx

```diff
@@ -70,21 +70,20 @@ export const FileUploadComponent = ({
         propertyPath={`${component.propertyPath}/properties/hasCustomFileEndings`}
       >
         {({ value }) => (
-          <LegacyRadioGroup
-            items={[
-              {
-                label: t('ux_editor.modal_properties_valid_file_endings_all'),
-                value: 'false',
-              },
-              {
-                label: t('ux_editor.modal_properties_valid_file_endings_custom'),
-                value: 'true',
-              },
-            ]}
+          <Radio.Group
             name={`${component.id}-valid-file-endings`}
-            variant='horizontal'
+            inline={true}
             value={value === true ? 'true' : 'false'}
-          />
+          >
+            {Children.toArray([
+              <Radio value='false' onChange={() => handleHasCustomFileEndingsChange('false')}>
```

Comment:
What do you think about using the value from the RadioButton itself, instead of hard-coding the value? 😊 
```suggestion
              <Radio value='false' onChange={(event) => handleHasCustomFileEndingsChange(event.target.value)}>
```

Reply:
Totally agree 😊, but I have passed event through arrow function because handleHasCustomFileEndingsChange deals with string and onChange expect event change

Reply:
Great! 👏